backend_new/test/unit/services/application.service.spec.ts

```diff
@@ -0,0 +1,516 @@
+      sendMailToMailingAddress: true,
+      householdExpectingChanges: true,
+      householdStudent: true,
+      incomeVouchers: true,
+      income: `income ${position}`,
+      incomePeriod: IncomePeriodEnum.perMonth,
+      preferences: {},
+      status: ApplicationStatusEnum.submitted,
+      submissionType: ApplicationSubmissionTypeEnum.electronical,
+      acceptedTerms: true,
+      submissionDate: date,
+      markedAsDuplicate: false,
+      confirmationCode: `confirmationCode ${position}`,
+      reviewStatus: ApplicationReviewStatusEnum.valid,
+      applicant: {
+        firstName: `application ${position} firstName`,
+        middleName: `application ${position} middleName`,
+        lastName: `application ${position} lastName`,
+        birthMonth: `application ${position} birthMonth`,
+        birthDay: `application ${position} birthDay`,
+        birthYear: `application ${position} birthYear`,
+        emailAddress: `application ${position} emailAddress`,
+        noEmail: false,
+        phoneNumber: `application ${position} phoneNumber`,
+        phoneNumberType: `application ${position} phoneNumberType`,
+        noPhone: false,
+        workInRegion: YesNoEnum.yes,
+        applicantWorkAddress: {
+          placeName: `application {i} applicantWorkAddress placeName`,
+          city: `application {i} applicantWorkAddress city`,
+          county: `application {i} applicantWorkAddress county`,
+          state: `application {i} applicantWorkAddress state`,
+          street: `application {i} applicantWorkAddress street`,
+          street2: `application {i} applicantWorkAddress street2`,
+          zipCode: `application {i} applicantWorkAddress zipCode`,
+          latitude: `${position}`,
+          longitude: `${position}`,
+        },
+        applicantAddress: {
+          placeName: `application {i} applicantAddress placeName`,
+          city: `application {i} applicantAddress city`,
+          county: `application {i} applicantAddress county`,
+          state: `application {i} applicantAddress state`,
+          street: `application {i} applicantAddress street`,
+          street2: `application {i} applicantAddress street2`,
+          zipCode: `application {i} applicantAddress zipCode`,
+          latitude: `${position}`,
+          longitude: `${position}`,
+        },
+      },
+      createdAt: date,
+      updatedAt: date,
+    };
+  };
+
+  const mockApplicationSet = (numberToCreate: number, date: Date) => {
+    const toReturn = [];
+    for (let i = 0; i < numberToCreate; i++) {
+      toReturn.push(mockApplication(i, date));
+    }
+    return toReturn;
+  };
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [ApplicationService, PrismaService],
+    }).compile();
+
+    service = module.get<ApplicationService>(ApplicationService);
+    prisma = module.get<PrismaService>(PrismaService);
+  });
+
+  it('testing list()', async () => {
```

Comment:
suggestion: update descriptions to be like "should test list endpoint"

Reply:
updated